.github/workflows/test-command.yml

```diff
@@ -149,6 +152,34 @@ jobs:
         repository: ${{ github.event.client_payload.pull_request.head.repo.full_name }}
         ref: ${{ github.event.client_payload.pull_request.head.ref }} 
 
+    # Determine requied version of terraform based on the target branch of the pull request. Then update the PATH to use it.
+    - name: "Determine required terraform version"
+      run: |
+        VERSION=$(cut -d/ -f1 <<<${BASE_REF})
+        if [[ ${VERSION} == 'master' ]]; then
+          VERSION=0.12
+        fi
```

Comment:
```suggestion
        if [[ ${VERSION} == 'master' ]] || [[ grep -qE 'required_version[^<]*0\.12' versions.tf ]]; then
          VERSION=0.12
        else
          VERSION=0.13
        fi
```

Reply:
Unfortunately `versions.tf` doesn't universally exist as I ran into this when mass-upgrading our modules. I'm fond for the simple explicit expression right now.  Master is 0.12 until we force master to be 0.13.

Reply:
I think a better long-term approach is to move this logic into `cloudposse/testing.cloudposse.co` and outside of this YAML. That way we can make it more robust. But shell scripting in YAML is not our favorite activity.

Reply:
We should just create a command (maybe Atlantis already did) that reads the Terraform, finds the latest acceptable version, and runs it. Call it `terraform-best-version` or something.